src/status_im2/contexts/wallet/common/network_preferences/view.cljs

```diff
@@ -4,59 +4,67 @@
             [quo.foundations.resources :as resources]
             [quo.theme :as quo.theme]
             [status-im2.contexts.wallet.common.network-preferences.style :as style]
-            [utils.i18n :as i18n]))
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
 
-(def mainnet
+(defn mainnet
+  [customization-color]
   [{:title        "Mainnet"
     :image        :icon-avatar
     :image-props  {:icon (resources/get-network :ethereum)
                    :size :size-20}
     :action       :selector
-    :action-props {:type :checkbox}}])
+    :action-props {:type                :checkbox
+                   :customization-color customization-color}}])
 
-(def networks-list
+(defn networks-list
```

Comment:
can we mark this as private? same for mainnet

Reply:
Yep, we can 👍